cypress/e2e/1-getting-started/todo.cy.js

```diff
@@ -12,132 +12,132 @@
 // https://on.cypress.io/introduction-to-cypress
 
 describe('example to-do app', () => {
```

Comment:
Please remove all automatically added examples. Keep your PR clean, it should contain only files needed for the review and related to the feature/fix.

Reply:
Thanks, removed all extra files